lib/fedora_migrate/rels_ext_datastream_mover.rb

```diff
@@ -4,26 +4,32 @@ class RelsExtDatastreamMover < Mover
     RELS_EXT_DATASTREAM = "RELS-EXT".freeze
 
     def migrate
-      statements.each do |statement|
-        subject.ldp_source.graph << [subject.rdf_subject, migrate_predicate(statement.predicate), migrate_object(statement.object)]
-      end
-      subject.ldp_source.update
+      migrate_statements
+      target.ldp_source.update
       update_index
     end
 
+    def post_initialize
+      @target ||= ActiveFedora::Base.find(source.pid.split(/:/).last)
```

Comment:
Some lines in this file reference `@target` and others `target`. Is this intentional?  Might it make sense to memoize `@target` in a method named `target` and then you can consistently reference the method rather than the ivar?

Is the effect of post_initialize basically to reload the target?